spec/models/person_spec.rb

```diff
@@ -603,4 +603,41 @@ def should_not_be_valid_swiss_post_code
 
   end
 
+  describe 'invalid address tags' do
```

Comment:
Gut das du das Verhalten in der Spec auflistest. Wenn ich das richtig verstehe, dann würde momentan das Tag gelöscht werden, wenn ich eine neue ungültige Adresse eingebe. Das sollte natürlich nicht so sein. Wahrscheinlich würde des sinn machen, das Tag auch beim speichern der person automatisch zu setzen (wenn ein dafür relevantes Attribute geändert wurde)

Reply:
Genau, aktuell wird das Tag gelöscht. Ich finde das aber grundsätzlich kein schlechter Weg. Bei der Email machen wir es auch so. Wir gehen grundsätzlich einfach davon aus, dass der Nutzer beim Abspeichern eine korrekte Addresse angibt. Wenn nicht, markiert das der Job und das Tag bleibt, bis die Person ihr Profil wieder updated.
Ich denke es wäre etwas komisch, wenn direkt nach dem Abspeichern einer Addresse, der Nutzer das Tag auf seinem Profil sieht, selbst wenn er eine korrekte Addresse angegeben hat (das würde ja passieren, wenn man das Tag automatisch immer setzt). Und wenn es dann erst beim Job gelöscht wird, würde es einfach ohne jeglichen Nutzerinput verschwinden. Das scheint für mich auch eher verwirrend zu sein.
Man könnte natürlich beim Abspeichern noch separat die Addresse validieren und basierend darauf das Tag setzen.

Reply:
Dh wenn ich ein Attribut ändere, das nichts mit der Adresse zu tun hat wird das Tag gelöscht. Ist das nicht noch 
verwirrender? Ich glaub die Kombination Validieren (wenn die Address Attribute ändern) und Tag setzten macht schon am meisten Sinn. Wir gehen ja davon aus, dass die Validierung nicht anschlägt und kein Tag gesetzt wird.